visualizer/validation.js

```diff
@@ -36,19 +36,19 @@ function validateNumber (num, targetDescription = '', conditions = {}) {
   return num
 }
 
-function uniqueMapKey (key, map) {
+function uniqueMapKey (key, map, separator) {
```

Comment:
It would be a `nice to have`, having them all start at `x1` rather than `x` for consistency and parsing simplicity

Reply:
Yeah I was thinking similar, I'll try it